docs/high/dataset.rst

```diff
@@ -261,6 +261,18 @@ dynamically loaded by the underlying HDF5 library. This is done by passing a
 filter number to :meth:`Group.create_dataset` as the ``compression`` parameter.
 The ``compression_opts`` parameter will then be passed to this filter.
 
+.. seealso::
+
+   `hdf5plugin <https://pypi.org/project/hdf5plugin/>`_
+     A Python package with several popular filters, including Blosc and LZ4,
```

Comment:
Q: which filters to highlight from hdf5plugin? I think Blosc should certainly be mentioned, and I picked LZ4 as the other one I think I'd heard of before learning about HDF5. But maybe it's better to mention ZFP (a lossy filter) to show the breadth. Or maybe it's simpler just to list them all.

cc @kif @vasole @t20100

Reply:
I'd mention Blosc as well and ZFP to have a lossy filter.
If you list it all, you probably want to do the same for HDF5 Filter Plugins, and we'll just have to think of updating `h5py` doc when we add a new filter to the zoo.